psyneulink/core/llvm/helpers.py

```diff
@@ -260,12 +260,26 @@ def get_condition_initializer(self, composition=None):
         return tuple(data)
 
     def bump_ts(self, builder, cond_ptr, count=(0, 0, 1)):
+        """
+        Increments the time structure of the composition.
+        Count should be a tuple where there is a number in only one spot, and zeroes elsewhere.
+        Indices greater than that of the one are zeroed.
+        """
+        
+        # Validate count tuple
+        count_set = False
+        for val in count:
+            if val != 0:
+                assert not count_set
+                count_set = True
```

Comment:
`assert count.count(0) == len(count) -1` should work as well and it's more readable